package.json

```diff
@@ -56,5 +60,6 @@
     "sorted-btree": "^1.8.1",
     "sucrase": "3.35.0",
     "yaml": "^2.3.3"
-  }
+  },
+  "packageManager": "yarn@1.22.19+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447"
```

Comment:
@GamerGirlandCo - did you mean to leave this here? It's a VERY specific version of yarn so I just wanted to be sure. :)

Reply:
yes, this is to ensure other package managers can't accidentally be used

Reply:
Makes sense.